### ISSUE TITLE
Mod needs to be updated for 1.21.2/1.21.3

### ISSUE THREAD

**Issue by EnKrypt** [squeek502/AppleSkin#333]
Fabric server with fabric client. The mod is installed on both server and client.
I'm using the latest versions of the fabric loader and the fabric API. If needed I can provide which versions.

When my client connects, I get this error in the client crashlog:
```
---- Minecraft Crash Report ----
// Shall we play a game?

Time: 2024-10-26 19:36:05
Description: Unexpected error

java.lang.NoSuchMethodError: 'float net.minecraft.class_1702.method_35219()'
	at knot//squeek.appleskin.client.HUDOverlayHandler.onPreRenderFood(HUDOverlayHandler.java:54)
	at knot//net.minecraft.class_329.handler$zzc000$appleskin$renderFoodPre(class_329.java:1538)
	at knot//net.minecraft.class_329.method_58477(class_329.java)
	at knot//net.minecraft.class_329.method_1760(class_329.java:874)
	at knot//net.minecraft.class_329.method_55805(class_329.java:555)
	at knot//net.minecraft.class_9080.method_55813(class_9080.java:36)
	at knot//net.minecraft.class_9080.method_55812(class_9080.java:22)
	at knot//net.minecraft.class_9080.method_55813(class_9080.java:36)
	at knot//net.minecraft.class_9080.method_55809(class_9080.java:29)
	at knot//net.minecraft.class_329.method_1753(class_329.java:234)
	at knot//net.minecraft.class_757.method_3192(class_757.java:531)
	at knot//net.minecraft.class_310.method_1523(class_310.java:1352)
	at knot//net.minecraft.class_310.method_1514(class_310.java:933)
	at knot//net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at knot//squeek.appleskin.client.HUDOverlayHandler.onPreRenderFood(HUDOverlayHandler.java:54)
	at knot//net.minecraft.class_329.handler$zzc000$appleskin$renderFoodPre(class_329.java:1538)
	at knot//net.minecraft.class_329.method_58477(class_329.java)
	at knot//net.minecraft.class_329.method_1760(class_329.java:874)
	at knot//net.minecraft.class_329.method_55805(class_329.java:555)
	at knot//net.minecraft.class_9080.method_55813(class_9080.java:36)
	at knot//net.minecraft.class_9080.method_55812(class_9080.java:22)
	at knot//net.minecraft.class_9080.method_55813(class_9080.java:36)
	at knot//net.minecraft.class_9080.method_55809(class_9080.java:29)
	at knot//net.minecraft.class_329.method_1753(class_329.java:234)

-- Uptime --
Details:
	JVM uptime: 49.283s
	Wall uptime: 43.487s
	High-res time: 41.142s
	Client ticks: 710 ticks / 35.500s
Stacktrace:
	at knot//net.minecraft.class_310.method_1587(class_310.java:2443)
	at knot//net.minecraft.class_310.method_54580(class_310.java:1002)
	at knot//net.minecraft.class_310.method_1514(class_310.java:957)
	at knot//net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Affected level --
Details:
	All players: 1 total; [class_746['EnKrypt'/4, l='ClientLevel', x=8.50, y=65.00, z=8.50]]
	Chunk stats: 961, 0
	Level dimension: minecraft:overworld
	Level spawn location: World: (8,64,8), Section: (at 8,0,8 in 0,4,0; chunk contains blocks 0,-64,0 to 15,319,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,-64,0 to 511,319,511)
	Level time: 0 game time, 0 day time
	Server brand: fabric
	Server type: Non-integrated multiplayer server
	Tracked entity count: 1

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: vanilla, fabric, appleskin, cloth-config, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-convention-tags-v2, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-loot-api-v3, fabric-message-api-v1, fabric-model-loading-api-v1, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v0, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, fallingleaves, iris, jade, modmenu, physicsmod, roughlyenoughitems, sodium, file/AVPBR Retextured R4.zip, file/Vanilla Mashup.zip

-- System Details --
Details:
	Minecraft Version: 1.21.3
	Minecraft Version ID: 1.21.3
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 21.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 374724688 bytes (357 MiB) / 1006632960 bytes (960 MiB) up to 2147483648 bytes (2048 MiB)
	CPUs: 16
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 7 7700X 8-Core Processor             
	Identifier: AuthenticAMD Family 25 Model 97 Stepping 2
	Microarchitecture: Zen 3
	Frequency (GHz): 4.50
	Number of physical packages: 1
	Number of physical CPUs: 8
	Number of logical CPUs: 16
	Graphics card #0 name: NVIDIA GeForce RTX 3070
	Graphics card #0 vendor: NVIDIA
	Graphics card #0 VRAM (MiB): 8192.00
	Graphics card #0 deviceId: VideoController1
	Graphics card #0 versionInfo: 32.0.15.6109
	Memory slot #0 capacity (MiB): 16384.00
	Memory slot #0 clockSpeed (GHz): 4.80
	Memory slot #0 type: Unknown
	Memory slot #1 capacity (MiB): 16384.00
	Memory slot #1 clockSpeed (GHz): 4.80
	Memory slot #1 type: Unknown
	Virtual memory max (MiB): 33897.99
	Virtual memory used (MiB): 16096.55
	Swap memory total (MiB): 2048.00
	Swap memory used (MiB): 8.40
	Space in storage for jna.tmpdir (MiB): available: 275614.59, total: 476291.53
	Space in storage for org.lwjgl.system.SharedLibraryExtractPath (MiB): available: 275614.59, total: 476291.53
	Space in storage for io.netty.native.workdir (MiB): available: 275614.59, total: 476291.53
	Space in storage for java.io.tmpdir (MiB): available: 275614.59, total: 476291.53
	Space in storage for workdir (MiB): available: 275614.59, total: 476291.53
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Fabric Mods: 
		appleskin: AppleSkin 3.0.5+mc1.21
		architectury: Architectury 14.0.3
		cloth-config: Cloth Config v16 16.0.141
			cloth-basic-math: cloth-basic-math 0.6.1
		fabric-api: Fabric API 0.106.1+1.21.3
			fabric-api-base: Fabric API Base 0.4.48+2127dc7240
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.78+c47b9d4340
			fabric-biome-api-v1: Fabric Biome API (v1) 14.0.5+c47b9d4340
			fabric-block-api-v1: Fabric Block API (v1) 1.0.28+c47b9d4340
			fabric-block-view-api-v2: Fabric BlockView API (v2) 1.0.16+c47b9d4340
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 2.0.2+c47b9d4340
			fabric-client-tags-api-v1: Fabric Client Tags 1.1.22+c47b9d4340
			fabric-command-api-v1: Fabric Command API (v1) 1.2.56+f71b366f40
			fabric-command-api-v2: Fabric Command API (v2) 2.2.35+c47b9d4340
			fabric-commands-v0: Fabric Commands (v0) 0.2.73+df3654b340
			fabric-content-registries-v0: Fabric Content Registries (v0) 9.1.5+c47b9d4340
			fabric-convention-tags-v1: Fabric Convention Tags 2.1.0+7f945d5b40
			fabric-convention-tags-v2: Fabric Convention Tags (v2) 2.9.0+c47b9d4340
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.3.3+c47b9d4340
			fabric-data-attachment-api-v1: Fabric Data Attachment API (v1) 1.1.38+c47b9d4340
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 21.0.14+c47b9d4340
			fabric-dimensions-v1: Fabric Dimensions API (v1) 4.0.7+c47b9d4340
			fabric-entity-events-v1: Fabric Entity Events (v1) 2.0.5+c47b9d4340
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 2.0.9+c47b9d4340
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.59+c47b9d4340
			fabric-item-api-v1: Fabric Item API (v1) 11.1.5+c47b9d4340
			fabric-item-group-api-v1: Fabric Item Group API (v1) 4.1.15+e7c9ab6640
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.53+c47b9d4340
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.51+df3654b340
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.3.22+c47b9d4340
			fabric-loot-api-v2: Fabric Loot API (v2) 3.0.24+3f89f5a540
			fabric-loot-api-v3: Fabric Loot API (v3) 1.0.12+c47b9d4340
			fabric-message-api-v1: Fabric Message API (v1) 6.0.19+c47b9d4340
			fabric-model-loading-api-v1: Fabric Model Loading API (v1) 3.0.7+c47b9d4340
			fabric-networking-api-v1: Fabric Networking API (v1) 4.3.3+56ec7ac640
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 17.0.7+c47b9d4340
			fabric-particles-v1: Fabric Particles (v1) 4.0.8+c47b9d4340
			fabric-recipe-api-v1: Fabric Recipe API (v1) 7.0.2+c47b9d4340
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 5.2.0+56ec7ac640
			fabric-renderer-api-v1: Fabric Renderer API (v1) 4.0.3+c47b9d4340
			fabric-renderer-indigo: Fabric Renderer - Indigo 1.9.3+c47b9d4340
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.54+73761d2e40
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.1.13+c47b9d4340
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.81+df3654b340
			fabric-rendering-v1: Fabric Rendering (v1) 8.0.5+c47b9d4340
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 5.0.8+c47b9d4340
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 3.0.5+c47b9d4340
			fabric-screen-api-v1: Fabric Screen API (v1) 2.0.32+c47b9d4340
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.98+c47b9d4340
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.29+c47b9d4340
			fabric-transfer-api-v1: Fabric Transfer API (v1) 5.4.0+c47b9d4340
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 6.1.8+c47b9d4340
		fabricloader: Fabric Loader 0.16.7
			mixinextras: MixinExtras 0.4.1
		fallingleaves: Falling Leaves 1.16.3
		iris: Iris 1.8.0-beta.6+mc1.21.2
			io_github_douira_glsl-transformer: glsl-transformer 2.0.1
			org_anarres_jcpp: jcpp 1.4.14
			org_antlr_antlr4-runtime: antlr4-runtime 4.13.1
		jade: Jade 16.0.4+fabric
		java: OpenJDK 64-Bit Server VM 21
		minecraft: Minecraft 1.21.3
		modmenu: Mod Menu 12.0.0-beta.1
			placeholder-api: Placeholder API 2.4.0-pre.2+1.21
		physicsmod: Physics Mod 3.0.21
		roughlyenoughitems: Roughly Enough Items 17.0.789
			error_notifier: Error Notifier 1.0.11
		sodium: Sodium 0.6.0-beta.3+mc1.21.2
	Loaded Shaderpack: ComplementaryUnbound_r5.3.zip
		Profile: VERYHIGH (+26 options changed by user)
	Launched Version: fabric-loader-0.16.7-1.21.3
	Launcher name: minecraft-launcher
	Backend library: LWJGL version 3.3.3-snapshot
	Backend API: NVIDIA GeForce RTX 3070/PCIe/SSE2 GL version 3.2.0 NVIDIA 561.09, NVIDIA Corporation
	Window size: 3440x1417
	GFLW Platform: win32
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Is Modded: Definitely; Client brand changed to 'fabric'
	Universe: 400921fb54442d18
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Render Distance: 12/12 chunks
	Resource Packs: vanilla, fabric, appleskin (incompatible), cloth-config, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-convention-tags-v2, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-loot-api-v3, fabric-message-api-v1, fabric-model-loading-api-v1, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v0, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, fallingleaves, iris, jade (incompatible), modmenu, physicsmod, roughlyenoughitems, sodium, file/AVPBR Retextured R4.zip (incompatible), file/Vanilla Mashup.zip (incompatible)
	Current Language: en_us
	Locale: en_US
	System encoding: Cp1252
	File encoding: UTF-8
	CPU: 16x AMD Ryzen 7 7700X 8-Core Processor 
```

The server crashes when I connect as well, and here's the crashlog from the server:
```
---- Minecraft Crash Report ----
// On the bright side, I bought you a teddy bear!

Time: 2024-10-26 19:36:03
Description: Ticking entity

java.lang.NoSuchMethodError: 'float net.minecraft.class_1702.method_35219()'
	at knot//squeek.appleskin.network.SyncHandler.onPlayerUpdate(SyncHandler.java:38)
	at knot//net.minecraft.class_3222.handler$zza000$appleskin$onUpdate(class_3222.java:2427)
	at knot//net.minecraft.class_3222.method_5773(class_3222.java)
	at knot//net.minecraft.class_3218.method_18762(class_3218.java:761)
	at knot//net.minecraft.class_1937.method_18472(class_1937.java:501)
	at knot//net.minecraft.class_3218.method_31420(class_3218.java:406)
	at knot//net.minecraft.class_5574.method_31791(class_5574.java:54)
	at knot//net.minecraft.class_3218.method_18765(class_3218.java:376)
	at knot//net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:1069)
	at knot//net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:953)
	at knot//net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:713)
	at knot//net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:292)
	at java.base/java.lang.Thread.run(Unknown Source)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
	at knot//squeek.appleskin.network.SyncHandler.onPlayerUpdate(SyncHandler.java:38)
	at knot//net.minecraft.class_3222.handler$zza000$appleskin$onUpdate(class_3222.java:2427)
	at knot//net.minecraft.class_3222.method_5773(class_3222.java)
	at knot//net.minecraft.class_3218.method_18762(class_3218.java:761)
	at knot//net.minecraft.class_1937.method_18472(class_1937.java:501)
	at knot//net.minecraft.class_3218.method_31420(class_3218.java:406)
	at knot//net.minecraft.class_5574.method_31791(class_5574.java:54)

-- Entity being ticked --
Details:
	Entity Type: minecraft:player (net.minecraft.class_3222)
	Entity ID: 4
	Entity Name: EnKrypt
	Entity's Exact location: -321.47, 63.00, -65.68
	Entity's Block location: World: (-322,63,-66), Section: (at 14,15,14 in -21,3,-5; chunk contains blocks -336,-64,-80 to -321,319,-65), Region: (-1,-1; contains chunks -32,-32 to -1,-1, blocks -512,-64,-512 to -1,319,-1)
	Entity's Momentum: 0.00, 0.00, 0.00
	Entity's Passengers: []
	Entity's Vehicle: null
Stacktrace:
	at knot//net.minecraft.class_1937.method_18472(class_1937.java:501)
	at knot//net.minecraft.class_3218.method_31420(class_3218.java:406)
	at knot//net.minecraft.class_5574.method_31791(class_5574.java:54)
	at knot//net.minecraft.class_3218.method_18765(class_3218.java:376)
	at knot//net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:1069)
	at knot//net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:953)
	at knot//net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:713)
	at knot//net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:292)
	at java.base/java.lang.Thread.run(Unknown Source)

-- Affected level --
Details:
	All players: 1 total; [class_3222['EnKrypt'/4, l='ServerLevel[world]', x=-321.47, y=63.00, z=-65.68]]
	Chunk stats: 841
	Level dimension: minecraft:overworld
	Level spawn location: World: (-16,95,16), Section: (at 0,15,0 in -1,5,1; chunk contains blocks -16,-64,16 to -1,319,31), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,-64,0 to -1,319,511)
	Level time: 45489 game time, 45489 day time
	Level name: world
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Commands: false
	Level weather: Rain time: 25100 (now: false), thunder time: 30498 (now: false)
	Known server brands: fabric
	Removed feature flags: 
	Level was modded: true
	Level storage version: 0x04ABD - Anvil
	Loaded entity count: 4
Stacktrace:
	at knot//net.minecraft.class_3218.method_8538(class_3218.java:1734)
	at knot//net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:1072)
	at knot//net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:953)
	at knot//net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:713)
	at knot//net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:292)
	at java.base/java.lang.Thread.run(Unknown Source)

-- System Details --
Details:
	Minecraft Version: 1.21.3
	Minecraft Version ID: 1.21.3
	Operating System: Linux (amd64) version 6.11.4-arch2-1
	Java Version: 23, Arch Linux
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Arch Linux
	Memory: 239580728 bytes (228 MiB) / 545259520 bytes (520 MiB) up to 8589934592 bytes (8192 MiB)
	CPUs: 12
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 5 1600 Six-Core Processor
	Identifier: AuthenticAMD Family 23 Model 1 Stepping 1
	Microarchitecture: Zen
	Frequency (GHz): -0.00
	Number of physical packages: 1
	Number of physical CPUs: 6
	Number of logical CPUs: 12
	Graphics card #0 name: GK208B [GeForce GT 710]
	Graphics card #0 vendor: NVIDIA Corporation (0x10de)
	Graphics card #0 VRAM (MiB): 160.00
	Graphics card #0 deviceId: 0x128b
	Graphics card #0 versionInfo: unknown
	Virtual memory max (MiB): 12066.89
	Virtual memory used (MiB): 2088.13
	Swap memory total (MiB): 4096.00
	Swap memory used (MiB): 0.00
	Space in storage for jna.tmpdir (MiB): <path not set>
	Space in storage for org.lwjgl.system.SharedLibraryExtractPath (MiB): <path not set>
	Space in storage for io.netty.native.workdir (MiB): <path not set>
	Space in storage for java.io.tmpdir (MiB): available: 7970.75, total: 7970.89
	Space in storage for workdir (MiB): available: 871570.31, total: 933764.06
	JVM Flags: 1 total; -Xmx8G
	Fabric Mods: 
		appleskin: AppleSkin 3.0.5+mc1.21
			cloth-config: Cloth Config v15 15.0.127
				cloth-basic-math: cloth-basic-math 0.6.1
		bluemap: BlueMap 5.4
			com_flowpowered_flow-math: flow-math 1.0.3
		collective: Collective 7.89
		craterlib: CraterLib 2.1.1
		fabric-api: Fabric API 0.106.1+1.21.3
			fabric-api-base: Fabric API Base 0.4.48+2127dc7240
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.78+c47b9d4340
			fabric-biome-api-v1: Fabric Biome API (v1) 14.0.5+c47b9d4340
			fabric-block-api-v1: Fabric Block API (v1) 1.0.28+c47b9d4340
			fabric-block-view-api-v2: Fabric BlockView API (v2) 1.0.16+c47b9d4340
			fabric-command-api-v1: Fabric Command API (v1) 1.2.56+f71b366f40
			fabric-command-api-v2: Fabric Command API (v2) 2.2.35+c47b9d4340
			fabric-commands-v0: Fabric Commands (v0) 0.2.73+df3654b340
			fabric-content-registries-v0: Fabric Content Registries (v0) 9.1.5+c47b9d4340
			fabric-convention-tags-v1: Fabric Convention Tags 2.1.0+7f945d5b40
			fabric-convention-tags-v2: Fabric Convention Tags (v2) 2.9.0+c47b9d4340
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.3.3+c47b9d4340
			fabric-data-attachment-api-v1: Fabric Data Attachment API (v1) 1.1.38+c47b9d4340
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 21.0.14+c47b9d4340
			fabric-dimensions-v1: Fabric Dimensions API (v1) 4.0.7+c47b9d4340
			fabric-entity-events-v1: Fabric Entity Events (v1) 2.0.5+c47b9d4340
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 2.0.9+c47b9d4340
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.59+c47b9d4340
			fabric-item-api-v1: Fabric Item API (v1) 11.1.5+c47b9d4340
			fabric-item-group-api-v1: Fabric Item Group API (v1) 4.1.15+e7c9ab6640
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.3.22+c47b9d4340
			fabric-loot-api-v2: Fabric Loot API (v2) 3.0.24+3f89f5a540
			fabric-loot-api-v3: Fabric Loot API (v3) 1.0.12+c47b9d4340
			fabric-message-api-v1: Fabric Message API (v1) 6.0.19+c47b9d4340
			fabric-networking-api-v1: Fabric Networking API (v1) 4.3.3+56ec7ac640
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 17.0.7+c47b9d4340
			fabric-particles-v1: Fabric Particles (v1) 4.0.8+c47b9d4340
			fabric-recipe-api-v1: Fabric Recipe API (v1) 7.0.2+c47b9d4340
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 5.2.0+56ec7ac640
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.54+73761d2e40
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.1.13+c47b9d4340
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 5.0.8+c47b9d4340
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 3.0.5+c47b9d4340
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.98+c47b9d4340
			fabric-transfer-api-v1: Fabric Transfer API (v1) 5.4.0+c47b9d4340
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 6.1.8+c47b9d4340
		fabricloader: Fabric Loader 0.16.7
			mixinextras: MixinExtras 0.4.1
		fallingtree: FallingTree 1.21.3.1
		fixedanvilrepaircost: Fixed Anvil Repair Cost 3.4
		jade: Jade 16.0.4+fabric
		java: OpenJDK 64-Bit Server VM 23
		minecraft: Minecraft 1.21.3
		sdlink: Simple Discord Link 3.2.0
		skinrestorer: SkinRestorer 2.1.0+1.21-fabric
		universal-graves: Universal Graves 3.5.0+1.21.2
			common-protection-api: Common Protection API 1.0.0
			fabric-permissions-api-v0: fabric-permissions-api 0.3.2-SNAPSHOT
			placeholder-api: Placeholder API 2.5.0+1.21.2
			polymer-core: Polymer (Core) 0.10.0+1.21.2
				polymer-networking: Polymer (Networking) 0.10.0+1.21.2
				polymer-registry-sync-manipulator: Polymer (Registry Sync Manipulator) 0.10.0+1.21.2
			polymer-resource-pack: Polymer (Resource Pack) 0.10.0+1.21.2
			polymer-virtual-entity: Polymer (Virtual Entity) 0.10.0+1.21.2
				polymer-common: Polymer (Common) 0.10.0+1.21.2
			predicate-api: Predicate API 0.6.0+1.21.2
			server_translations_api: Server Translations API 2.4.0+1.21.2-rc1
				packet_tweaker: Packet Tweaker 0.6.0-pre.1+1.21.2-pre3
			sgui: sgui 1.7.1+1.21.2
	Server Running: true
	Player Count: 1 / 10; [class_3222['EnKrypt'/4, l='ServerLevel[world]', x=-321.47, y=63.00, z=-65.68]]
	Active Data Packs: vanilla, fabric, collective, fabric-convention-tags-v2, server_translations_api, universal-graves
	Available Data Packs: collective, fabric, fabric-convention-tags-v2, minecart_improvements, redstone_experiments, server_translations_api, trade_rebalance, universal-graves, vanilla, winter_drop
	Enabled Feature Flags: minecraft:vanilla
	World Generation: Stable
	World Seed: -889395136709676447
	Suppressed Exceptions: ~~NONE~~
	Is Modded: Definitely; Server brand changed to 'fabric'
	Type: Dedicated Server (map_server.txt)
```

**Comment by ghost** [comment]
Thats probably because the mod isnt updated to 1.21.3 yet.

**Comment by EnKrypt** [comment]
@squeek502 Just checking. Have you had the chance to look into this?

**Comment by squeek502** [comment]
Yes, working on it in this branch: https://github.com/squeek502/AppleSkin/tree/1.21.3-fabric

Need to test it and then get the NeoForge version updated as well.

**Comment by squeek502** [comment]
Released as v3.0.6.
